src/components/context/authContext.tsx

```
@@ -1,127 +1,144 @@
-import { DEFAULT_USER } from '@/types/const';
-import { UserProps } from '@/types/interface';
-import { createContext, useContext, useEffect, useState } from 'react';
+"use server"
+
+import { DEFAULT_USER } from "@/types/const";
+import { UserProps } from "@/types/interface";
+import { createContext, useContext, useEffect, useState } from "react";
 import { jwtDecode } from 'jwt-decode';
 import axios from 'axios';
 import Cookies from 'js-cookie';
+import { fa, fi } from "@faker-js/faker";
```

ESLint

[error] 'fa' is defined but never used.

(@typescript-eslint/no-unused-vars)